ansible.cfg

```diff
@@ -4,8 +4,8 @@ display_skipped_hosts = false
 show_per_host_start = true
 nocows = 1
 inventory = $PWD/inventories/
-roles_path = ./playbooks/galaxy_roles:./user_playbooks/roles:./roles
-collections_paths = ./playbooks/galaxy_collections
+roles_path = .cache/galaxy_collections:./user_playbooks/roles:./roles
```

Comment:
I don't think that setting `roles_path` to something …collections is the right thing. There are no roles inside that folder, only collections (that might contain roles, but that is not relevant for the "roles_path".